code_samples/product_specific/pycharm_basics/README.md

```diff
@@ -0,0 +1,24 @@
+# PyCharm Sample [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [PyCharm Plugin Development in IntelliJ SDK Docs][docs:pycharm]*
+
+## Quickstart
+
+PyCharm Sample is a plugin that depends on the PyCharm IDE having the proper dependencies specified
+in the Gradle configuration file. Implementation utilizes a simple action added to the *MainMenu* group displaying
```

Comment:
Suggest: _The PyCharm Sample plugin demonstrates the correct Gradle project configuration for developing plugins targeting the PyCharm IDE. A simple action class is implemented to demonstrate adding an entry to the PyCharm IDE *MainMenu* group._